jupyter-notebooks/analysis-ready-data/ard_2_use_case_1.ipynb

```diff
@@ -143,7 +188,7 @@
     "    ]\n",
     "}\n",
     "\n",
-    "### Let's search:\n",
+    "# Let's search:\n",
     "# for the geometry above\n",
```

Comment:
Maybe make these criteria part of the notebook markdown text rather than comments? Just so they stand out and you can bullet them properly.